data_ocean/utils.py

```diff
@@ -0,0 +1,44 @@
+"""
+Here we are collecting alf functions for cleaning xml records from whitespaces and /
+abbreviations, splitting tag.texts and converting strings with date info to datetimes
+"""
+import datetime
+import re
+
+
+# changing to lowercase, deleting 'р.', 'м.', 'с.', 'смт.', 'смт', 'сщ.', 'с/рада.', 'сщ/рада.', /
+# 'вул.' from a string
+def clean_name(name):
+    name = name.lower()
+    return re.sub(r'р\.|м\.|с\.|смт|смт\.|сщ\.|с/рада\.|сщ/рада\.', "", name).trim()
```

Comment:
What is `trim` function???
```suggestion
    return re.sub(r'р\.|м\.|с\.|смт|смт\.|сщ\.|с/рада\.|сщ/рада\.', "", name.lower()).strip()
```

Reply:
Sorry, heritage of Java learning(